esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3261,39 +3169,32 @@ bool CFileSprayEx::onDropZoneFiles(IEspContext &context, IEspDropZoneFilesReques
         if (dropZoneList.ordinality())
             resp.setDropZones(dropZoneList);
 
-        if (netAddressStr.length() < 1)
-            return true;
-
-        char pathSep = '/';
-        if (osStr && *osStr)
-        {
-            int os = atoi(osStr);
-            if (os == OS_WINDOWS)
-                pathSep = '\\';
-        }
+        const char* dzName = req.getDropZoneName();
```

Comment:
Is this optional? Does this break backward compatibility ?
e.g. if old client talks to this new esp without this dropzone name..

How does the called getDropZoneFiles() cope if blank ?